projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -505,4 +505,58 @@ public function add_gutenberg_menus( $wp_admin = false ) {
 	public function should_link_to_wp_admin() {
 		return get_user_option( 'jetpack_admin_menu_link_destination' );
 	}
+
+	/**
+	 * Returns the current slug from the URL.
+	 *
+	 * @return string
+	 */
+	public function get_current_slug() {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		global $pagenow;
+		$slug = $pagenow;
+		if ( isset( $_REQUEST['post_type'] ) ) {
+			$slug .= '?post_type=' . $_REQUEST['post_type'];
+		}
+		if ( isset( $_REQUEST['taxonomy'] ) ) {
+			$slug .= '?taxonomy=' . $_REQUEST['taxonomy'];
+		}
+		if ( isset( $_REQUEST['page'] ) ) {
+			$slug .= '?page=' . $_REQUEST['page'];
+		}
+
+		return $slug;
+	}
+
+	/**
+	 * Prepend a dashboard swithcer to the "Screen Options" box of the current page.
+	 * Callback for the 'screen_settings' filter (available in WP 3.0 and up).
+	 *
+	 * @param string $current The currently added panels in screen options.
+	 *
+	 * @return string The HTML code to append to "Screen Options"
+	 */
+	public function register_dashboard_switcher( $current ) {
+		include_once __DIR__ . '/menu-mappings.php';
```

Comment:
instead of ```include_once __DIR__ . '/menu-mappings.php';``` we could replace it with something like ``` $menu_mappings = require __DIR__ . '/menu-mappings.php';```

Reply:
That's black magic to me! You are my hero @BogdanUngureanu ! 
4799b92

cc: @tjcafferkey

Reply:
Love this. TIL thanks @BogdanUngureanu